testsuite/tests/tool-ocamldoc/Alerts.html.reference

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+<link rel="stylesheet" href="style.css" type="text/css">
+<meta content="text/html; charset=iso-8859-1" http-equiv="Content-Type">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="Start" href="index.html">
+<link rel="Up" href="index.html">
+<link title="Index of values" rel=Appendix href="index_values.html">
+<link title="Index of modules" rel=Appendix href="index_modules.html">
+<link title="Alerts" rel="Chapter" href="Alerts.html"><title>Alerts</title>
+</head>
+<body>
+<div class="navbar">&nbsp;<a class="up" href="index.html" title="Index">Up</a>
+&nbsp;</div>
+<h1>Module <a href="type_Alerts.html">Alerts</a></h1>
+
+<pre><span id="MODULEAlerts"><span class="keyword">module</span> Alerts</span>: <code class="code"><span class="keyword">sig</span></code> <a href="Alerts.html">..</a> <code class="code"><span class="keyword">end</span></code></pre><div class="info module top">
+<div class="info-desc">
+<p>Test the rendering of alerts.</p>
+</div>
+</div>
+<hr width="100%">
+
+<pre><span id="VALa"><span class="keyword">val</span> a</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-desc">
+<p>Documentation after.</p>
+</div>
+<ul class="info-attributes">
+<li><b>Alert foo.</b> bar</li>
+</ul>
+</div>
+
+<pre><span id="VALb"><span class="keyword">val</span> b</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-desc">
+<p>Documentation before.</p>
+</div>
+<ul class="info-attributes">
+<li><b>Alert foo.</b> bar</li>
+</ul>
+</div>
+
+<pre><span id="VALc"><span class="keyword">val</span> c</span> : <code class="type">int</code></pre><div class="info ">
+<ul class="info-attributes">
+<li><b>Alert foo.</b> bar</li>
+</ul>
+</div>
+<p>The deprecated attribute.</p>
+
+<pre><span id="VALd"><span class="keyword">val</span> d</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-deprecated">
+<span class="warning">Deprecated. </span>foo</div>
+</div>
+
+<pre><span id="VALd'"><span class="keyword">val</span> d'</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-deprecated">
+<span class="warning">Deprecated. </span>foo</div>
+</div>
+<p>No payload</p>
+
+<pre><span id="VALe"><span class="keyword">val</span> e</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-deprecated">
+<span class="warning">Deprecated. </span></div>
+</div>
+
+<pre><span id="VALf"><span class="keyword">val</span> f</span> : <code class="type">int</code></pre><div class="info ">
+<ul class="info-attributes">
+<li><b>Alert foo.</b></li>
+</ul>
+</div>
+<p>Both deprecated tag and alert</p>
+
+<pre><span id="VALg"><span class="keyword">val</span> g</span> : <code class="type">int</code></pre><div class="info ">
+<div class="info-deprecated">
+<span class="warning">Deprecated. </span>foo
+bar</div>
```

Comment:
We have
```
<span class="warning">Deprecated. </span>foo
bar</div>
```
where I expected

```
<span class="warning">Deprecated. </span>foo</div>
```
See the comment above.

Reply:
I don't know how I could miss this! Perhaps some premature refactoring.